Makefile

```
@@ -26,6 +26,19 @@ build-go: create-bin-dir
 		-o ${BINARY_DIR}/${BINARY_NAME} \
 		.
 
+build-go-cover: create-bin-dir
+	go mod download
+	export CURRENT_DIR=$(shell pwd); \
+	CC=gcc \
+	CGO_CFLAGS="-I $$CURRENT_DIR/libbpfgo/output" \
```

I'm surprised by the `$$`

I somehow hate Makefile, they are not fun buddies to play with.

I would have expected to see


```suggestion
	CGO_CFLAGS="-I ${CURRENT_DIR}/libbpfgo/output" \
```

---

this doesn't take effect when is surrounded by `""`

---

Ok, thanks for checking.

I'm curious what could report a project like unmake

https://github.com/mcandre/unmake